addons/mail/__manifest__.py

```diff
@@ -142,6 +142,8 @@
             'mail/static/src/core/common/**/*',
             'mail/static/src/core/web_portal/**/*',
             'mail/static/src/core/web/**/*',
+            'mail/static/src/chatter/web_portal/**/*',
+            'mail/static/src/chatter/web/**/*',
```

Comment:
I'm surprised to see these added again, didn't we make the new bundle/moves in a prep PR?